decidim-proposals/app/controllers/decidim/proposals/admin/proposals_controller.rb

```diff
@@ -14,6 +14,10 @@ class ProposalsController < Admin::ApplicationController
         helper_method :proposals, :query, :form_presenter, :proposal, :proposal_ids
         helper Proposals::Admin::ProposalBulkActionsHelper
 
+        def index
+          check_admin_session_filters
```

Comment:
IMHO this method call is a bit confusing? Could it be added as a `before_action`?

Reply:
Maybe could be a before action in the concern Filterable?

Reply:
You are right, I added the method inside the index method explicitly because there is no definition of the index action in the controller or included concerns, but the `before_action` works fine.

About adding the `before_action` filter in the concern it would affect to all the filters of admin, but I'm not sure if the idea is to extend this behavior to all of them in this issue or only affect proposals for the moment

Reply:
Ouch, there was a lint offense I did not remember, index action has to be explicitly defined in the class to use the filter

```
Offenses:

decidim-proposals/app/controllers/decidim/proposals/admin/proposals_controller.rb:17:9: C: Rails/LexicallyScopedActionFilter: index is not explicitly defined on the class.
        before_action :check_admin_session_filters, only: [:index]
```

Reply:
Yes, the linter is warning you. Maybe leave the method empty?

Reply:
It worked